web/package.json

```diff
@@ -67,6 +67,7 @@
     "react-responsive-carousel": "^3.2.23",
     "react-select": "^5.7.0",
     "react-table": "^7.8.0",
+    "sentry": "^0.1.2",
```

Comment:
This probably doesn't need to be installed - could you try following https://docs.sentry.io/platforms/javascript/guides/nextjs/ for the installation?